tests/testthat/test-qenv_concat.R

```diff
@@ -1,21 +1,23 @@
 testthat::test_that("Concatenate two identical qenvs outputs", {
-  env <- new.env()
-  env$iris1 <- iris
-  q1 <- new_qenv(quote(iris1 <- iris), env = env)
+  q <- qenv()
+  q1 <- eval_code(q, quote(iris1 <- iris))
   q2 <- q1
 
   q <- concat(q1, q2)
```

Comment:
To avoid re-definition

```suggestion
  q0 <- qenv()
  q1 <- eval_code(q0, quote(iris1 <- iris))
  q2 <- q1

  q <- concat(q1, q2)
```